dcc-submission-ui/dictionary/app/scripts/helpers/dictionary-utils.js

```diff
@@ -1,4 +1,4 @@
-/* globals $ */
+/* globals $,jsondiffpatch */
```

Comment:
You can use `$window.jsondiffpatch` instead in an angular context. See my previous comment.